deployment/scripts/sonarqube.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+echo "SonarQube Scan"
+echo "Directory: $PWD"
+echo "JAVA_HOME: $JAVA_HOME"
+
+mkdir $PWD/sonarqube/
```

Comment:
Here and throughout the script, any variables like `$PWD` that define filesystem paths need to be quoted like `"$PWD"` (or ideally like `"${PWD}"` also to prevent potential variable resolution issues).

Variables in paths _should always_ be quoted to minimize the risk of unexpected characters blowing up your commands. For example, if any directory in the PWD stack contains a space in its name, if you don't quote the variable, the space character will break the one command argument into two parts, and bad things could happen.